drivers/serial/uart_nrfx_uart.c

```diff
@@ -346,6 +346,18 @@ static int uart_nrfx_init(struct device *dev)
 			       CONFIG_UART_0_NRF_TX_PIN,
 			       CONFIG_UART_0_NRF_RX_PIN);
 
+#ifdef CONFIG_UART_0_NRF_FLOW_CONTROL
+	(void) gpio_pin_configure(gpio_dev,
```

Comment:
(void) , not necessary.

Reply:
Won't there be some Lint complaining about unused return value?

Reply:
fixed